<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1087</width>
    <height>811</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>900</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>HdTrader v0.0</string>
  </property>
  <property name="unifiedTitleAndToolBarOnMac">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>221</width>
      <height>271</height>
     </rect>
    </property>
    <property name="title">
     <string>잔고</string>
    </property>
    <widget class="QLabel" name="label_account">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>64</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>계좌</string>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBox_account">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>30</y>
       <width>121</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QTableWidget" name="tableWidget_jango">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>70</y>
       <width>201</width>
       <height>191</height>
      </rect>
     </property>
     <property name="inputMethodHints">
      <set>Qt::ImhNone</set>
     </property>
     <property name="lineWidth">
      <number>0</number>
     </property>
     <property name="columnCount">
      <number>1</number>
     </property>
     <attribute name="horizontalHeaderVisible">
      <bool>false</bool>
     </attribute>
     <attribute name="horizontalHeaderCascadingSectionResizes">
      <bool>false</bool>
     </attribute>
     <attribute name="horizontalHeaderDefaultSectionSize">
      <number>120</number>
     </attribute>
     <attribute name="verticalHeaderDefaultSectionSize">
      <number>30</number>
     </attribute>
     <row>
      <property name="text">
       <string>예수금</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>총매입</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>총평가</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>총손익</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>총수익률</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>추정자산</string>
      </property>
     </row>
     <column/>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>20</y>
      <width>761</width>
      <height>561</height>
     </rect>
    </property>
    <property name="title">
     <string>보유종목현황 및 포트폴리오 목록</string>
    </property>
    <widget class="QTableWidget" name="tableWidget_stock">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>731</width>
       <height>192</height>
      </rect>
     </property>
     <attribute name="horizontalHeaderDefaultSectionSize">
      <number>120</number>
     </attribute>
     <column>
      <property name="text">
       <string>종목명</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>보유량</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>매입가</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>현재가</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>평가손익</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>수익률</string>
      </property>
     </column>
    </widget>
    <widget class="QCheckBox" name="checkBox_check_balance_real">
     <property name="geometry">
      <rect>
       <x>540</x>
       <y>230</y>
       <width>111</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>실시간 조회</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_check_balance">
     <property name="geometry">
      <rect>
       <x>660</x>
       <y>230</y>
       <width>93</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>조회</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_file_load">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>230</y>
       <width>93</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>File Load</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_3">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>300</y>
      <width>221</width>
      <height>261</height>
     </rect>
    </property>
    <property name="title">
     <string>수동주문</string>
    </property>
    <widget class="QLabel" name="label_hoga">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>120</y>
       <width>64</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>종류</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="spinBox_price">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>180</y>
       <width>161</width>
       <height>22</height>
      </rect>
     </property>
     <property name="maximum">
      <number>100000000</number>
     </property>
    </widget>
    <widget class="QLabel" name="label_order">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>64</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>주문</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_num">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>150</y>
       <width>64</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>수량</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_order">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>220</y>
       <width>93</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>현금주문</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_code">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>64</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>종목</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_code">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>60</y>
       <width>161</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_name">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>90</y>
       <width>161</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(170, 255, 255);</string>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBox_order">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>30</y>
       <width>161</width>
       <height>22</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>신규매수</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>신규매도</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>매수취소</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>매도취소</string>
      </property>
     </item>
    </widget>
    <widget class="QLabel" name="label_price">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>180</y>
       <width>64</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>가격</string>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBox_hoga">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>120</y>
       <width>161</width>
       <height>22</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>시장가</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>지정가</string>
      </property>
     </item>
    </widget>
    <widget class="QSpinBox" name="spinBox_num">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>150</y>
       <width>161</width>
       <height>22</height>
      </rect>
     </property>
     <property name="maximum">
      <number>100000</number>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1087</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
